main.go

```diff
@@ -322,9 +326,9 @@ func getClientIP(r *http.Request) (string, error) {
 	return xRealIP, nil
 }
 
-func updateBalance(c *rpcclient.Client) {
+func updateBalance(c *dcrwallet.Client) {
 	// calculate balance
-	gbr, err := c.GetBalanceMinConf(cfg.WalletAccount, 0)
+	gbr, err := c.GetBalanceMinConf(context.TODO(), cfg.WalletAccount, 0)
```

Comment:
If `updateBalance` accepts context as a param, the request context can be passed when it is called from a web request.

Fine to pass `context.TODO()` when `updateBalance` is called from `main()`.